content/artwork/index.html.haml

```diff
@@ -1,13 +1,57 @@
 ---
-layout: default
+layout: simplepage
 title: Jenkins Artwork
+tight: true
+notitle: true
```

Comment:
Looking good. We don't need `notitle` here, we can just remove the `h1` from this page and let `simplepage` display the title.